app/code/core/Mage/GoogleAnalytics/Block/Ga.php

```diff
@@ -398,14 +397,16 @@ protected function _getOrdersTrackingCodeAnalytics4()
                     'item_id' => $_product->getSku(),
                     'item_name' => $_product->getName(),
                     'price' => number_format($_product->getFinalPrice(), 2),
+
+                    'quantity' => intval($productInCart->getQty()),
```

Comment:
Cast to `int` to follow #2872

```suggestion
                    'quantity' => (int) $productInCart->getQty(),
```

Reply:
sorry i have not read refering to pr #2872

Reply:
you have tested with (int)?

Reply:
It's the same thing functionally

Reply:
yes tested with php 74 ok thanks